### PR TITLE
Implement and plot all figures

### PR DESCRIPTION
**Pull request by universmile**

### CODE REVIEW

File: examples/world3/figures/capital.jl [hunk 1]
```diff
@@ -25,7 +25,7 @@ fig_3_36_variables = [
     (ag.aiph,  0, 50,    "aiph"),
 ]
 
-plotvariables(sol, (t, 1900, 1970), fig_3_36_variables, name="Fig. 3.36", showlegend=true, showaxis=true, colored=true)
+plotvariables(sol, (t, 1900, 1980), fig_3_36_variables, name="Fig. 3.36", showlegend=true, showaxis=true, colored=true)
```
Comment (natema) [inline examples/world3/figures/capital.jl:28]: @universmile, Paulo says you fixed this but you still have to update this PR then.

Comment (universmile) [reply examples/world3/figures/capital.jl]: @natema, the PR has been updated.

**Comment by natema** [comment]
It is fine for me to merge into `dev`.

File: examples/world3/figures/world3.jl [hunk 2]
```diff
@@ -88,3 +147,175 @@ system = world3_historicalrun(agriculture_tables=agr_tables_7_13)
...
+nr_parameters_7_38 = World3.NonRenewable.getparameters()
+nr_parameters_7_38[:nruf2] = 0.125
+
+pol_parameters_7_38 = World3.Pollution.getparameters()
+pol_parameters_7_38[:ppgf21] = 0.25
+
+system = world3_historicalrun(pop_params=pop_parameters_7_37, capital_params=cap_parameters_7_38, nonrenewable_params=nr_parameters_7_38, pollution_params=pol_parameters_7_38, capital_tables=cap_tables_7_36, agriculture_tables=agr_tables_7_36)
+sol_7_38 = WorldDynamics.solve(system, (1900, 2100))
+
+plotvariables(sol_7_38, (t, 1900, 2100), fig_7_7_variables, name="Fig. 7.38", showlegend=true, showaxis=true, colored=true)
```
Comment (paulobruno) [inline examples/world3/figures/world3.jl:321]: It's missing a blank line at the end of the file. But since it was already merged, we can fix this later.

**Pull request merged**